src/Routes/Router.js

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+import { Routes, Route } from "react-router-dom";
+import { Landing, Login, Signup } from "../components/components";
```

Comment:
Instead of storing the components from a components.js file you can export it from an index.js file too that lies within the components folder so that when you try to import the components it can be written as 
import {Landing,Login,Signup} from "./components";

Reply:
Nice suggestion, implemented